### DISCUSSION TITLE
深夜歌曲推薦

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1850] [General]
現在是深夜時間，很適合emo，跟大家分享幾首最近喜歡在晚上聽的歌曲(其實是想emo的時候會聽的)。我聽歌比較看重歌詞，有感覺的歌詞很容易打動我，但旋律也很重要XD

1.<如果我們不曾相遇> 五月天
這首MV其實是在述說愛情，但我每次都是代入國、高中同學和親人，因為我很懷念國、高中的時光和相處起來很可愛的同學，(雖然大學也不錯，但就是不免更懷念從前哈哈哈)可能跟他們經歷過很多事吧，所以情感相對而言比較深厚。以下這三部分是我最喜歡的，每次聽到都會狠狠地共情住。

"如果我們不曾相遇 你又會在哪裡？如果我們從不曾相識 人間又如何運行？"
"那一天 那一刻 那個場景 你出現在我生命 每一分 每一秒 每個表情 故事都充滿驚奇"
"某一天 某一刻 某次呼吸 我們終將再分離 而我的自傳裡曾經有你"

2.<飛> 林宥嘉
這首歌非常小眾，觀看次數才100萬(抖音同名歌曲竟然有1000萬)，是我前幾天不小心挖到的XD ，旋律非常溫柔、很適合平復情緒，聽完真的是很平靜。如果你有奮鬥追夢或者努力堅持做自己喜歡的事的經歷，或許會跟歌詞有共鳴。雖然我聽的是翻唱版本，但翻唱者有相關的人生故事，所以我自己聽起來更帶感、更能代入。

"哪個朋友還沒笑過我的夢
哪一種安慰 不會讓我更痛
還有多少關於放棄的理由
能理直氣壯 背後沒有惶恐

這個世界不缺偉大和成功
而我的世界也不怕失敗者
何不讓我追求 我的快樂

還沒學會飛 天那麼蔚藍
為何迎面的風對我一直牽絆
我笨拙地飛 是很怕孤單
卻怕收起翅膀會更遺憾"

3.<有生之年> 張遠
這算是一首表達祝願的歌吧，我們所希望的、想擁有的，幾乎在這首歌裡了。看歌詞可能感覺不大，但配上歌手的聲線、旋律和真誠，是真的會被打動(我自己啦，但要看歌手的真誠度哈哈哈，這是最重要的元素) 尤其在不順遂、不如意的時候聽，會想哭XD

"有生之年 願你沒有遺憾
  有生之年 願你笑容燦爛"
"願你千杯不醉總會有良人相伴
 獨具慧眼看透這變幻人間
 願你歷經時間 仍是少年
 願你累了倦了有人會為你分擔
 每個夜晚都會有美夢作伴
 願你長路漫漫 得償所願"

好啦，該睡覺了，大家晚安zz 
打卡3：53

**Comment by sorn32** [comment]
如果是深夜歌單的話，推薦
漂鳥 - 理想混蛋雞丁
I'm More Sober When I'm Drunk - 韋禮安的音樂專輯
我覺得都很適合(*´∀`)~♥

**Reply by thz0000** [reply]
韋的英文專輯是真的很好聽><最喜歡R.I.P！
理想混蛋的話滿喜歡行星的~

**Reply by sorn32** [reply]
我也是最喜歡R.I.P (˶‾᷄ ⁻̫ ‾᷅˵) 
突然想到，習慣不習慣也不錯，我還蠻常聽的

**Comment by 12Jesse24** [comment]
如果有在聽饒舌的話，這邊推楊和蘇的<從那天起>、<讓我快樂>，雖然是建議分手的時候聽的，因為裡面歌詞很多關於情侶故事XD
，不過還是蠻推的深夜emo歌。

**Reply by woohsin** [reply]
楊和蘇得"很久很久"真的是emo神曲，句句punchline

**Comment by TobywithDino** [comment]
我最近很愛聽The Lumineers的歌曲，在深夜聽著他們歌，就會開始想像自己在美國的大路上開始公路旅行，心情非常的好，也不會太EMO。
也可以選擇專注聽樂器，我覺得他們的吉他和鼓的聲音根本無敵。

**Comment by chiuwangwang** [comment]
我最近喜歡上了法語歌，雖然都聽不懂，推薦幾首
Yamê - Bécane: 這首歌非常適合CHILL 的時候聽，感覺就很飄
Black M - French Kiss:饒舌，節奏感很強
Le Festin- Camille:料理鼠王的主題曲，有時候想聽，就去看整部電影，真的很配

**Comment by woohsin** [comment]
Halsey的一首 3am超社和晚上聽，然後發現自己跟歌詞裡的一樣孤單 ;)

**Comment by jeff110303549** [comment]
這邊大推胡凱兒，這個團體就是我在大一的時候踏進聽團的第一步，歌曲風格都偏向抒情，而且每一首歌所帶來的體驗都完全不一樣，這邊容許我私心推薦幾首：
<停靠>：有關於對過去的回憶和對某人的思念，從往事中尋找那些像要表達的情緒，不論時間過了多久、也不論未來將會如何，都仍願意為了這份心意而奮不顧身。
<如何>：一首描述對於生活的迷茫與內心困惑的歌，歌曲中對人生的意義投出質疑：追逐意義的過程是否具有意義。如果你也對現如今的生活有所疑惑，不如試試和這首歌憶起抒發心情。
<菸癮>：是一首無論如何都不能錯過的歌，傾訴出了每一個對於生活感到不安而又無可奈何的人，那些藏在心底、不知道如何釋放的陳悶；整首歌沒有一句提到菸，卻如同吸了一口菸後，將內心的複雜感受吐出口中，只留下一縷飄向遠方的煙。

**Comment by YifenLin** [comment]
這裡推薦一首呂允<你睡了之後>，不知道甚麼時候被推到這首就愛上了，在晚上一個人的時候聽會讓心情特別平靜，很喜歡呂允的聲音，有種鄰家男孩的感覺。

**Comment by tzurou** [comment]
推安溥<最好的時光>，整首歌都是簡單的吉他和弦，配上安溥特有的療癒型聲線，還有最讓我有感觸的歌詞，很適合自己一個人獨處時聽，像是道盡了人的一生必定會經歷的事，也終將釋懷。

**Comment by SusuTima** [comment]
推薦 
莫文蔚-《這世界那麼多人》
梁靜茹-《情歌》
宇宙人-《藍色的你》
怕胖團-《月旁月光》

**Comment by sunset513** [comment]
我這裡的推薦比較特別，有各式曲風的曲目清單。

1. 西洋流行歌的話，我很喜歡Script的《Rain》

每每聽到都會想到過往的回憶，我的眼淚都會像雨滴滴落。

2. 中文老歌的話，我不得不推薦伍佰版本的《夢醒時分》

歌詞戳到我的心坎，「早知道傷心總是難免的，你又何苦一往情深」，再加上伍佰老師的嗓音，不知道出了多少人的心情。

3. 古典：貝多芬的悲愴奏鳴曲

從第一樂章聽到第三樂章，我的心情會從各種情緒交雜而出的悲痛，漸漸放開，最終釋懷。

**Comment by woohsin** [comment]
又來蹭分數。
今天就決定是 yurri 優里了。
ベテルギウス(參宿四)，他的聲音很適合一個人躲起來偷偷聽，這首剛好用星星作為主題(雖然我覺得星星的意涵沒有很重要)很適合晚上聽，其實他所有的歌都很適合~~

**Comment by yxin214** [comment]
必須推一首棒球大聯盟的心繪，雖然聽過無數次我還是沒真正搞懂每一句歌詞的意思，但光聽旋律就很有振奮人心的作用，而且意外發現有看過棒球大聯盟的人都很喜歡這首歌！

**Comment by EPHMR0524** [comment]
日文歌的話大推R Sound Design 的帝國少女和flos，慵懶的旋律配上略為憂傷的歌詞，是挺適合深夜emo時聽的，如果聽不慣軟體(初音未來)唱的版本作者自己也有翻唱，聽起來又會是另一種感覺，情感上更為豐富。
阿對了，還有須田景凪的猫被り和 パメラ也很不錯

**Comment by 20050201Shawn** [comment]
深夜想要emo的時候我都聽joji的歌，像是afterthought跟glimpes of us，他比較像是lofi饒舌風，歌詞都蠻抑鬱。或聽一些很傷心的鋼琴，剛看完期中成績我要去聽歌emo了。

**Comment by yyujimmy** [comment]
這些歌曲確實都充滿了深度和情感，讓人在深夜時刻靜心聆聽，思考人生、感受情感的起伏。歌詞的深度和旋律的溫柔都讓人沉浸其中，帶來一種溫暖和慰藉的感覺。

對於《如果我們不曾相遇》，你將其代入到國、高中時的回憶中，這樣的情感共鳴確實讓歌曲更具意義和深度。每次聆聽，都能感受到那段美好的時光和與重要人物的情感聯繫。

《飛》和《有生之年》也都是那種能夠鼓舞人心、讓人思考人生意義的歌曲。特別是當面對挑戰和困難時，這些歌曲更能給人力量和勇氣，讓我們堅持追夢、珍惜當下。

謝謝你分享這些歌曲，它們不僅讓你得到了情感上的寄託，也讓我們一起感受到音樂的力量。晚安，願你有個美好的夢。
